src/pages/Profiles.vue

```diff
@@ -650,15 +603,16 @@ export default class Profiles extends Vue {
     }
 
     async updateProfileList() {
-        this.profileList = ["Default"];
         const profilesDirectory: string = this.activeProfile.getDirectory();
         await fs.readdir(profilesDirectory).then(dirContents => {
-            dirContents.forEach(async (file: string) => {
-                if ((await fs.stat(path.join(profilesDirectory, file))).isDirectory() && file.toLowerCase() !== 'default' && file.toLowerCase() !== "_profile_update") {
-                    this.profileList = [...this.profileList, file].sort();
-                }
+            let promises = dirContents.map(async function(file) {
+                return ((await fs.stat(path.join(profilesDirectory, file))).isDirectory() && file.toLowerCase() !== 'default' && file.toLowerCase() !== "_profile_update")
+                    ? file : undefined;
             });
-        }).catch(() => { /* Do nothing */ });
```

Comment:
first commit adds the whole updateProfileList()`, `with just `return;`, later one converts to this commented style, soooo... better not touch it, because it _may_ be critical?

Reply:
Hard to say since we don't know if the error it's intended to silence is something we actually want to silence. I assume you haven't been able to get this part to cause any erros? If so, we can probably go with this approach and show the error modal. We can always yeet it if we get reports of users getting unnecessary modals when the error could just be ignored.

Reply:
Yep, let's go with that. 👍